python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -454,6 +454,26 @@ class ScarletDeblendConfig(pexConfig.Config):
         doc="Names of flags which should never be deblended."
     )
 
+    # Testing options
+    # Some obs packages and ci packages run the full pipeline on a small
+    # subset of data to test that the pipeline is functioning properly.
+    # This is not meant as scientific validation, so it can be useful
+    # to only run on a small subset of the data that is large enough to
+    # test the desired pipeline features but not so long that the deblender
+    # is the tall pole in terms of build times.
```

Comment:
I'd say "test execution time" rather than "build time". This isn't changing build times as such.